CMakeLists.txt

```diff
@@ -77,8 +78,9 @@ set (pomerol_sources
     pomerol/EnsembleAverage
 )
 
-configure_file(${CMAKE_SOURCE_DIR}/include/pomerol.h.in ${CMAKE_BINARY_DIR}/include/pomerol.h)
```

Comment:
Igor, do you remember the reason of splitting `first_include.h.in` and `pomerol.h.in` ?

Reply:
At this point I can only guess, and my guess is that including `first_include.h` gives you access to all those configuration macros without including the rest of pomerol headers.

I am going to remove this `configure_file()` as there is nothing to configure in `pomerol.h.in`.